### ISSUE TITLE
Add EE authentication to CI

### ISSUE THREAD

**Issue by alxmrs** [google/Xee#53]
This should fix our CI.

**Comment by alxmrs** [comment]
@giswqs do you know of a good way to add EE authentication within a Github Action? What does geemap do?

**Comment by giswqs** [comment]
Here is what I did: create a GitHub reposity secret with the name `EARTHENGINE_TOKEN`. Open the engine engine credentials file (`.config/earthengine/credentials`) and copy its entire content and set it as the value of the GitHub reposity secret `EARTHENGINE_TOKEN`.

Then you can copy this `ee_initialize()` function from geemap to xee and use it to authenticate EE.
https://github.com/gee-community/geemap/blob/master/geemap/common.py#L34

Basically, the function reads the GitHub environment variable `EARTHENGINE_TOKEN` and saves the credentials file in the file path `.config/earthengine/credentials`. 

Similar to what I did for interactive web apps
https://huggingface.co/spaces/giswqs/solara-geemap

**Comment by naschmitz** [comment]
This is done. CI is working properly.
